cdap-docs/examples-manual/source/examples/sport-results.rst

```diff
@@ -0,0 +1,212 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform WordCount Application
+    :copyright: Copyright © 2014-2015 Cask Data, Inc.
+
+.. _examples-fileset:
+
+=============
+Sport Results
+=============
+
+A Cask Data Application Platform (CDAP) Example demonstrating partitioned file sets through sport results analytics.
+
+Overview
+========
+
+This Application demonstrates the use of the PartitionedFileSet dataset:
+
+   - Game results are stored in the PartitionedFileSet ``results``. It is partitioned by league and season,
```

Comment:
These bulleted points should not be indented but instead flush left. (The lines after the first are indented, but the entire block should be moved to the left margin.) Common.